modules/Hoot/managers/api.js

```diff
@@ -936,7 +936,27 @@ export default class API {
         };
 
         return this.request( params )
-            .then( resp => this.statusInterval( resp.data.jobid ) );
+            .then( resp => this.statusInterval( resp.data.jobid ) )
+            .then( resp => {
+                return {
+                    data: resp.data,
+                    message: 'Clip job complete',
+                    status: 200,
+                    type: resp.type
+                };
+            } )
+            .catch( err => {
+                let message, status, type;
+
+                status = err.status;
+
+                if ( status >= 500 ) {
+                    message = 'Error during Clip! Please try again later.';
```

Comment:
we should use the technique from the jobs panel branch (1331) to get the `stderr` value here